src/remotestorage.js

```diff
@@ -763,8 +763,12 @@
     },
 
     localStorageAvailable: function () {
```

Comment:
Yes, but can't we just access the feature list in memory then? Isn't that initializing every all features on load? I.e. if the feature is loaded, then save.